buildspec.yml

```diff
@@ -14,6 +14,9 @@ phases:
     commands:
       - echo Testing the newly built Docker image...
       - docker run --name test -d -e RAILS_ENV=test $CONTAINER_NAME:test /bin/bash -c "tail -f /dev/null"
+      - git config --global credential.helper store
+      - echo "https://$GIT_USERNAME:$GIT_PASSWORD@github.com" >
+        /root/.git-credentials
```

Comment:
Do you need this? It's a public repo, so anyone should be able to pull from it.

Reply:
issue is at Docker level it can't clone tenk repo which is set to private even after adding dxw-dalmatian as a user. 
through speaking to bob he said there might be other configuration which needs to be done at container level.

Therefore, I don't know if this is still required after docker is told about it inside the container as bob mentioned in previous comment in this commit

Reply:
Yeah, we're using a private fork of Tenk. The way I do it in https://github.com/dxw/breathe_to_10k is by setting a `BUNDLE_GITHUB__COM` environment variable, which is an access token with access to private repos. See https://github.com/dxw/breathe_to_10k/blob/master/.github/workflows/run.yml

Reply:
That's a nice solution @pezholio. I think username and password is probably fine for this for now, though.